packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/review/load-testing.api.md

```diff
@@ -687,7 +687,7 @@ export class TestProfileAdministrationOperationsClient {
     createOrUpdateTestProfile(testProfileId: string, body: TestProfileAdministrationOperationsClientTestProfile, options?: CreateOrUpdateTestProfileOptionalParams): Promise<TestProfileAdministrationOperationsClientTestProfile>;
     deleteTestProfile(testProfileId: string, options?: DeleteTestProfileOptionalParams): Promise<void>;
     getTestProfile(testProfileId: string, options?: GetTestProfileOptionalParams): Promise<TestProfileAdministrationOperationsClientTestProfile>;
-    listTestProfiles(options?: ListTestProfilesOptionalParams): TestProfileAdministrationOperationsClientPagedAsyncIterableIterator<TestProfileAdministrationOperationsClientTestProfile>;
+    listTestProfiles(options?: ListTestProfilesOptionalParams): PagedAsyncIterableIterator<TestProfileAdministrationOperationsClientTestProfile>;
```

Comment:
This is an example where the test case is only existing in smoke testing but not covered in UT or cadl-ranch.